ajax/libs/bootstrap-drawer/1.0.5/js/drawer.js

```diff
@@ -0,0 +1,213 @@
+/* ========================================================================
+ * Bootstrap: drawer.js v1.0.3
```

Comment:
Um...
This line said the version is v1.0.3, but you need v1.0.5.
I think you need to confirm this with the author.